src/coreclr/src/CMakeLists.txt

```diff
@@ -60,6 +60,12 @@ if(CLR_CMAKE_HOST_UNIX)
   add_subdirectory(nativeresources)
 endif(CLR_CMAKE_HOST_UNIX)
 
+# Workaround for errors on GCC
+if ((CMAKE_CXX_COMPILER_ID STREQUAL "GNU") AND (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 10.0))
+    add_compile_options(-Wno-error=stringop-overflow=)
+    add_compile_options(-Wno-error=tautological-compare)
```

Comment:
This is also disabled in `eng\native\configurecompiler.cmake` for clang. I would make it disable for centrally, for all compilers in `eng\native\configurecompiler.cmake`. This is useless warning.